src/Testing/QueueHelper.php

```diff
@@ -37,55 +37,86 @@ public function __construct(ContainerInterface $container)
     /**
      * Initialize destinations for message reception
      *
-     * @param string|null $connectionName
-     * @param string $queue
+     * @param string $destination
      *
      * @return $this
      */
-    public function init(string $connectionName = null, string $queue = 'queue'): QueueHelper
+    public function init(string $destination = null): QueueHelper
     {
-        $connection = $this->connection($connectionName);
+        if ($destination === null) {
+            @trigger_error("Since 1.2: the destination parameter can not be null. Will be removed in 2.0", \E_USER_DEPRECATED);
+        }
```

Comment:
Add the deprecation on `else` of the condition `if (func_num_args() > 1) {`, because in case of the using legacy interface a deprecation will already be triggered.